exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -355,35 +386,15 @@ func (ku *Kucoin) UpdateTicker(ctx context.Context, p currency.Pair, assetType a
 
 // UpdateTickers updates all currency pairs of a given asset type
 func (ku *Kucoin) UpdateTickers(ctx context.Context, assetType asset.Item) error {
+	var errs error
 	switch assetType {
 	case asset.Futures:
-		ticks, err := ku.GetFuturesOpenContracts(ctx)
+		ticks, err := ku.GetFuturesTickers(ctx)
```

Comment:
Although this is technically correct calling the futures ticker endpoint, we have sacrificed efficiency here for the benefit of two fields (Ask and Bid) and another which is less clear (Size is the last traded volume, rather than the previous 24Hr volume which is more useful), whilst losing High and Low. I'd advocate for still using `GetFuturesOpenContracts` which is one API request and a person can use the new wrapper function if they really want Bid and Ask. Or they could just fetch the orderbooks for their desired pairs. A lot of the newly opened PRs pertain to ticker batch optimisations to reduce slowness if you have a lot of pairs enabled

Reply:
@thrasher- Completely agreed on efficiency sacrifice here, and it's an awful trade off.

However the issue is that High/Low/Size are all for 24 hours.
I don't think we can justifiably use 24 hour data for a ticker that isn't 1 day. It's dangerously misleading, isn't it?
What we *could* do is to reject any time period other than 1D and then we can use `GetFuturesOpenContracts` guilt free.

Thoughts?

Reply:
24 hour data rolling values are less noisy and more useful as a trader, which the API docs mention those values are all using that time window. Bid/Ask for the blue chips are easy to calculate even with missing data since you could do something like (last/mark/index price - min quote step size or - min exchange fee pct) and that would get you pretty close to the actual values since market markers would be attempting to profit off the spread. 

The issue here is both of efficiency and loss of more useful fields. As a trader and in terms of priority of info, I'd expect some equivalence of Last/Index/Mark price, 24hr volume, open interest and any permutation of High, Low, Close over the past 24 hrs. Bid/Ask is better left for orderbook fetching since by the time you call the ticker, the bid/ask pricing and size would of changed. For someone who doesn't have websocket enabled and wants a ticker snapshot, batching + this info is still far superior. Spawning tonnes of go routines to compensate then runs you up against the exchanges rate limit when we didn't really need to do that

Reply:
Okay. Understood. Thanks for the explanation.

So we leave `GetTickers` using `GetFuturesOpenContracts` and take what other improvements are left on the cutting board after that, including, as you say, the new wrapper function for `GetFuturesTickers`

Reply:
Yup 👍

Reply:
I have updated the function